src/sentry/snuba/referrer.py

```diff
@@ -846,6 +846,9 @@ class Referrer(Enum):
     TAGSTORE_GET_TAG_VALUE_PAGINATOR_FOR_PROJECTS = "tagstore.get_tag_value_paginator_for_projects"
     TASKS_MONITOR_RELEASE_ADOPTION = "tasks.monitor_release_adoption"
     TASKS_PERFORMANCE_SPLIT_DISCOVER_DATASET = "tasks.performance.split_discover_dataset"
+    TASKS_PERFORMANCE_SPLIT_DISCOVER_DATASET_METRICS_ENHANCED = (
+        "tasks.performance.split_discover_dataset.metrics-enhanced.primary"
+    )
```

Comment:
No harm, but as of https://github.com/getsentry/sentry/pull/73734/ you don't have to add `.primary` at the end! Any defined referrer with `".primary"` and `".secondary"` referrers are all considered valid 👍🏻